src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -22,39 +25,54 @@ export default function WebexRemoteMedia({className, meetingID}) {
     remoteShare,
     error,
   } = useMeeting(meetingID);
+  const members = useMemberships(meetingID, 'meeting');
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
   const shareRef = useStream(remoteShare);
+  const hasOtherMembers = members.length > 1;
   const hasMedia = !!(remoteAudio || remoteVideo || remoteShare);
   const hasTwoMedia = remoteVideo && remoteShare;
   const mainClasses = {
     [`${WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media`]: true,
     [className]: !!className,
     'split-screen': hasTwoMedia,
   };
+  let remoteDisplay;
 
-  return (
-    <div className={classNames(mainClasses)}>
-      {error ? (
-        <AlertBanner show type="warning">
-          Having trouble joining the meeting? Please check your connection.
-        </AlertBanner>
-      ) : (
-        <>
-          {!hasMedia ? (
-            <Badge rounded>
-              <Spinner size={16} />
-              <div>Connecting</div>
-            </Badge>
-          ) : null}
+  if (error) {
+    remoteDisplay = (
+      <AlertBanner show type="warning">
+        Having trouble joining the meeting? Please check your connection.
+      </AlertBanner>
+    );
+  } else if (hasMedia && hasOtherMembers) {
+    remoteDisplay = (
+      <>
+        {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
 
-          {remoteVideo ? <video ref={videoRef} playsInline autoPlay /> : null}
+        {remoteShare ? <video ref={shareRef} playsInline autoPlay /> : null}
 
-          {remoteShare ? <video ref={shareRef} playsInline autoPlay /> : null}
+        {remoteAudio ? <audio ref={audioRef} autoPlay /> : null}
+      </>
+    );
+  } else if (hasMedia && !hasOtherMembers) {
+    remoteDisplay = (
+      <h4>
+        <div>Waiting for others to join...</div>
```

Comment:
Headings are block-type elements. The `div` is not necessary in this case
```suggestion
        Waiting for others to join...
```